core/server/api/configuration.js

```diff
@@ -27,7 +28,7 @@ function getBaseConfig() {
         fileStorage:    config.get('fileStorage') !== false,
         useGravatar:    !config.isPrivacyDisabled('useGravatar'),
         publicAPI:      config.get('publicAPI') === true,
-        blogUrl:        config.get('url').replace(/\/$/, ''),
```

Comment:
A project search shows me that we are also using `config.get('urlSSL')` without the url helper.
Shouldn't we use `utils.url.urlFor('home', {secure: true}, true)` in this case?